apps/fabric-website/src/styles/_base.scss

```diff
@@ -62,8 +73,13 @@
   }
 
   // Feedback section styles
-  .FeedbackList-button .ms-Button-label { // Feedback button color overridden by UHF
+  .FeedbackList-button .ms-Button-label {
+    // Feedback button color overridden by UHF
     @include ms-fontColor-white;
+
+    @include high-contrast {
+      color: window;
```

Comment:
@jdhuntington This is potentially super-nit but aren't system colors usually pascal case? If this doesn't cause an issue then disregard.

Reply:
Good call; that does appear to be correct. (TIL!)